### DISCUSSION TITLE
Select a Player on Navigate

### DISCUSSION THREAD

**Discussion by lou-cucci** [punxaphil/custom-sonos-card#563] [Ideas]
Hi Phil

Awesome work!

I have the below badge for each player that takes me to my Sonos Page when selected. 

Is there a way that when I navigate to the Sonos card i can select the entity to display. 

For example, I select the Living Room Sonos badge and it jumps to the Sonos Page (that already happens) but when it lands on the page it selects the Living Room Sonos so I can see and control what is playing there

Thanks

Lou 

`type: custom:mushroom-template-badge
entity: media_player.living_room_tv_area
label: Living Room Sonos
icon: |
  {% if states('media_player.living_room_tv_area') != 'playing' %}
    mdi:speaker-off
  {% else %}
    mdi:speaker
  {% endif %}
color: |
  {% if states('media_player.living_room_tv_area') != 'playing' %}
    blue
  {% else %}
    green
  {% endif %}
content: |
  {{ states('media_player.living_room_tv_area') | capitalize  }}
tap_action:
  action: navigate
  navigation_path: /lovelace/sonos
`

**Comment by punxaphil** [comment]
Linking to specific player: https://github.com/punxaphil/custom-sonos-card?tab=readme-ov-file#linking-to-specific-player


---
PunxaPhil
[![image](https://github.com/user-attachments/assets/cb2e240e-f9ca-4900-a425-b8089216c1ee)](https://patreon.com/PunxaPhil) [![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)

**Comment by lou-cucci** [comment]
Worked

Rockstar thanks!
